sympy/vector/vector.py

```diff
@@ -40,6 +40,34 @@ def components(self):
         # subclass of Vector the instance belongs to.
         return self._components
 
+    @property
+    def projections(self):
```

Comment:
The difference is that in the output we inform user which components are equals to `0`. Sometimes it's convenient. In `components` we can't rely on order of returned value because elements are cut when projection on base vector is `0`